test/extensions/filters/network/zookeeper_proxy/filter_test.cc

```diff
@@ -30,7 +31,19 @@ class ZooKeeperFilterTest : public testing::Test {
   ZooKeeperFilterTest() { ENVOY_LOG_MISC(info, "test"); }
 
   void initialize() {
-    config_ = std::make_shared<ZooKeeperFilterConfig>(stat_prefix_, 1048576, scope_);
+    // Populate latency threshold for Multi opcode. The latency thresholds of all other opcode
+    // fallback to the default threshold. Since we did not specify the default threshold, it
+    // fallbacks to 100 millisecond.
+    auto* threshold = thresholds.Add();
+    threshold->set_opcode(
+        envoy::extensions::filters::network::zookeeper_proxy::v3::LatencyThreshold::Multi);
+    threshold->mutable_threshold()->set_nanos(200000000); // 200 millisecond
+
+    Protobuf::RepeatedPtrField<
+        envoy::extensions::filters::network::zookeeper_proxy::v3::LatencyThreshold>&
```

Comment:
I think we can use `LatencyThresholdList` here and there instead of full name

Reply:
I'm confused why we need a ref here. and tailing underscore is the naming style of data member.

Reply:
Sorry we do not need a ref here and pass the value to the func directly.